tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -35,6 +35,9 @@ spec:
     description: "aws eks enpoint to create clusters against"
   - name: cluster-name
     description: The name of the EKS cluster you want to spin.
+  results:
+    - name: datapoint
+      description: Stores the cl2 loadtest result to emit cw metrics to aws cloudwatch   
```

Comment:
description: Stores the cl2 loadtest result to be consumed by others (e.g. emitting to CloudWatch)